imagej/imagej-ops2-tutorial/docs/CallingOps.md

```diff
@@ -0,0 +1,66 @@
+# Calling Ops
+
+Ops are designed to be called from the Op matcher, using the `OpBuilder` syntax - this page describes that syntax in detail.
+
+On this page, we will be constructing an `OpBuilder` call on an `OpEnvironment ops` to execute a Gaussian Blur on an input image `inImage`, with our output buffer `outImage`.
+
+# Specifying the name with `.op()`
+
+From the `OpEnvironment`, an `OpBuilder` call is initialized with the method `OpEnvironment.op(String)`, which is used to describe the name of the Op that the `OpBuilder` must return:
+
+```groovy
+ops.op("filter.gauss")
+```
+
+# Specifying the arity with `.arityX()`
+
+After providing the name, the `OpBuilder` syntax requires the number of inputs. If you intend to pass `X` inputs to the Op, you then chain the `arityX()` method:
+
```

Comment:
I hesitate to answer that directly, because I'm hoping it's obvious from the wiki page :sweat_smile: 

Please tell me your understanding after reading the change made in bb7aa138, @elevans!

Reply:
This is great and I see my misunderstanding. Can you add "...we do this by adding on of the `arityX()` methods, **where `X` is the number of inputs**, to chain...".

I know this is pretty minor but the `code` syntax of `arityX()` actually made me think there is an `arityX()` that you can use!

Reply:
Maybe `arity*` would be clearer, since `*` is not a valid character for method names.

Reply:
> Maybe `arity*` would be clearer, since `*` is not a valid character for method names.

That's a great idea - I did it with 0e461ce8